<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>firstQtClass</class>
 <widget class="QMainWindow" name="firstQtClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>firstQt</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>12163291 강현지</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_upload">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>upload</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cvtgray">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>gray</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cvthsv">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>hsv</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cvtycrcb">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>240</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ycrcb</string>
    </property>
   </widget>
   <widget class="QLabel" name="img_input">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>161</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>input</string>
    </property>
   </widget>
   <widget class="QLabel" name="img_gray">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>0</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="img_hsv">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>110</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="img_ycrcb">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>230</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="firstQt.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_upload</sender>
   <signal>clicked()</signal>
   <receiver>firstQtClass</receiver>
   <slot>push_upload()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>116</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>pushButton()</slot>
  <slot>push_upload()</slot>
 </slots>
</ui>
